### DISCUSSION TITLE
Where is 3.2.x pulsar-client-kafka?

### DISCUSSION THREAD

**Discussion by dev-moonduck** [apache/pulsar#22849] [Q&A] [answered]
According to the official documentation,
```xml
<dependency>
  <groupId>org.apache.pulsar</groupId>
  <artifactId>pulsar-client-kafka</artifactId>
  <version>3.2.3</version>
</dependency>
```
I don't see this version in maven central.2.11.x is the last version there. Do you support latest version of this that match with current latest version of Pulsar?

**Comment by lhotari** [comment, marked as answer]
It is managed in https://github.com/apache/pulsar-adapters and released separately. There haven't been releases since 2.11.0 . It doesn't follow the Pulsar releases, but it should continue to be compatible with recent Pulsar client versions. 

To use a newer Pulsar client version, it's recommended to use the Pulsar BOM. Please see docs at https://pulsar.apache.org/docs/3.2.x/client-libraries-java-setup/#pulsar-bom how to use Pulsar BOM. pulsar-client-kafka is a Kafka compatibility wrapper over the Pulsar Java client. You might also need [to use the unshaded version](https://pulsar.apache.org/docs/3.2.x/adaptors-kafka/#use-the-pulsar-kafka-compatibility-wrapper-together-with-existing-kafka-client) of `pulsar-client-kafka` which has the artifact id `pulsar-client-kafka-original` .

**Reply by dev-moonduck** [reply]
Thank you. I think the official documentation need to be fixed

**Reply by lhotari** [reply]
I agree. Would you like to contribute an improvement? The repository for docs is https://github.com/apache/pulsar-site . There are some instructions in https://pulsar.apache.org/contribute/document-contribution/ and https://pulsar.apache.org/contribute/document-preview/ for working with the documentation updates.
